bsip-0061.md

```diff
@@ -0,0 +1,53 @@
+    BSIP: 0061
+    Title: Fix locked accounts with circular dependencies 
+    Authors: OpenLedgerApp <https://github.com/OpenLedgerApp>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-22-02
+    Updated: 
+    Discussion: https://github.com/bitshares/bsips/issues/94
+    Worker:
+
+
+# Abstract
+BitShares allow to change permissions of account A to account A. See *Cycles* paragraph of https://bitshares.org/technology/dynamic-account-permissions for details. More information can be found here: https://steemit.com/blockchain/@hipster/sad-story-how-i-lost-bitshares-account
```

Comment:
The Abstract should contain a synopsis of the problem and proposed solution. An explanation of cycles is needed here to explain what happened in the past to allow accounts to become locked and how they will be prevented in the future with this change. 

References to supporting documentation should be placed in the "See Also" section at the end of this document.

Reply:
Done.